provpatterns/assets/transform-morphology-atlas-space-prov-template.svg

```diff
@@ -1,2 +1,2 @@
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
I suggest to use nsg:Reconstructionfromimage as a reconstruction activity and not nsg:Reconstrcution

Reply:
Done